docs/tools/cli/service.rst

```diff
@@ -312,12 +374,52 @@ Service Privatelink commands
 ``avn service queries``
 '''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
 
-List service query statistics
+Lists the service connections/queries statistics for an Aiven for PostgreSQL or Aiven for MySQL. 
+The list of queries data points retrievable includes:
+
+* the ``public.pg_stat_statements`` `columns <https://www.postgresql.org/docs/current/pgstatstatements.html>`_, for Aiven for PostgreSQL services.
+* the ``performance_schema.events_statements_summary_by_digest`` `columns <https://dev.mysql.com/doc/refman/8.0/en/performance-schema-statement-summary-tables.html>`_, for Aiven for MySQL services.
+
+A description of the retrieved columns for Aiven for PostgreSQL can be found in the dedicated `PostgreSQL documentation <https://www.postgresql.org/docs/current/pgstatstatements.html>`_ .
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``service_name``
+    - The name of the service
+  * - ``--fromat``
+    - The format string for output defining the query metrics to retrieve, e.g. `'{calls} {total_time}'` 
```

Comment:
Markup weirdness here when I look at this in preview. Maybe should be double backtick to give code-formatted output that includes the single quotes?

Reply:
yep done